src/components/DatePicker/CalendarHeader/__tests__/CalendarHeader.spec.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { CalendarHeader } from '../CalendarHeader';
+
+describe('CalendarHeader', () => {
+    const MONTH_NAMES = [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December',
+    ];
+    const YEARS = [2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025];
+    const decreaseMonth = jest.fn();
+    const increaseMonth = jest.fn();
+    const changeYear = jest.fn();
+    let wrapper;
+
+    beforeEach(() => {
+        wrapper = mount(
+            <CalendarHeader
```

Comment:
I think the whole point of this component is the styling, so that the apps don't have to do it as well. Why would we want to allow changes to a small part of the calendar?